tests/integration/testdata/start_api/swagger-rest-api-template.yaml

```diff
@@ -61,6 +61,13 @@ Resources:
           - image/gif
       StageName: prod
     Type: AWS::ApiGateway::RestApi
+  Dev:
+    Type: AWS::ApiGateway::Stage
```

Comment:
Is this used in the integ tests? I don't see any tests that were updated or added that use this template.